### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by Warfields**
Workers Weekly release

### CODE REVIEW

**Review by kflansburg — APPROVED** [review]

**Comment by Warfields** [comment]
CC: @irvinebroque

File: content/workers/platform/changelog.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ rss: file
 
 # Changelog
 
+## 2023-07-14
+- The Node.js util.MIMEType API is now available for workers using the `nodejs_compat` compatibility flag
```
Comment (irvinebroque) [inline content/workers/platform/changelog.md]:
```suggestion
- An implementation of the [util.MIMEType](https://nodejs.org/api/util.html#class-utilmimetype) API from Node.js is now available when the [`nodejs_compat` compatibility flag](https://developers.cloudflare.com/workers/runtime-apis/nodejs/) is enabled
```

@jasnell — let's make sure https://developers.cloudflare.com/workers/runtime-apis/nodejs/util/ gets updated to reflect

**Review by irvinebroque — APPROVED** [review]
🚀

File: content/workers/platform/changelog.md (hunk 1 above)
Comment (irvinebroque) [reply content/workers/platform/changelog.md]: (going to make a pr now, 1 sec)

Comment (irvinebroque) [reply content/workers/platform/changelog.md]: https://github.com/cloudflare/cloudflare-docs/pull/9879

**Pull request merged**